formats/rbxlx.md

```diff
@@ -82,7 +82,7 @@ Attribute                       | Value
 - **Child of:** [`<roblox>`][roblox]
 - **Contains:** text
 
-Maps a metadata key to a value. Only present in `rbxm` files.
+Maps a metadata key to a value. Only present in `rbxmx` files.
```

Comment:
Would be better to say "Only present in model files" to avoid giving the impression that it's present in `rbxmx`, but not `rbxm`.